mozilla/blurts-server
# MNTOR-2265: Skip first onboarding step when clicking “Start free scan”

# References: 
Jira: [MNTOR-2265](https://mozilla-hub.atlassian.net/browse/MNTOR-2265)



# Description

When users are coming from the resolution flow to the onboarding: Directly send them to the “Enter info” step.

# How to test

Note: The steps below will be only fully testable after we implement [MNTOR-2264](https://mozilla-hub.atlassian.net/browse/MNTOR-2264). Instead of navigating to the second step of the “Welcome flow” via the CTAs the essence of this PR can be tested by visiting directly `/redesign/user/welcome/free-scan`.

1. Make sure your user:
    - Has no OneRep profile
    - Has not run a scan
    - Has no scan results
    - Is not subscribed to Premium
    - Is a returning user
3. Visit `/redesign/user/dashboard` and `/redesign/user/dashboard/fix/data-broker-profiles/start-free-scan`
4. Click “Start free scan”
5. You should be directed to the “Enter info” step of the Premium welcome onboarding

[MNTOR-2265]: https://mozilla-hub.atlassian.net/browse/MNTOR-2265?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[MNTOR-2264]: https://mozilla-hub.atlassian.net/browse/MNTOR-2264?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
